toolbox/db/db_add_subject.m

```diff
@@ -67,7 +67,7 @@
 sqlConn = sql_connect();
 
 % Check the subject unicity
-if sql_row_exists(sqlConn, 'Subject', struct('Name', sSubject.Name))
+if sql_query(sqlConn, 'exist', 'Subject', [], struct('Name', sSubject.Name))

```

Comment:
In which situation would you need the 4th argument?

Reply:
For now the 4th argument can be anything, it is discarded to match the input arguments in `sql_query`.
It could be used to verify the existence of at least N number of rows that match the condition

Reply:
If that were the case it should be an optional argument, whereas the conditional structure is mandatory (in most cases). I would change the order of the arguments so that this call becomes:

`sql_query(sqlConn, 'exist', 'Subject', struct('Name', sSubject.Name))`

Reply:
Yes, that'd solve it